docs/Getting_Started.md

```diff
@@ -0,0 +1,437 @@
+
+---
+
+## Overview
+
+In this guide, we’ll be building a Discord app that lets server members play a *slightly*-enhanced version of rock paper scissors (with 7 choices instead of the usual 3).
+
+### Resources used in this guide
+- **[Github repository](https://github.com/shaydewael/discord-getting-started)** where the code from this guide lives along with additional feature-specific examples.
+- **[discord-interactions](https://github.com/discord/discord-interactions-js)**, an npm package which provides types and helper functions for Discord apps.
+- **[Glitch](https://glitch.com/)**, an online development environment to build and host apps. You could also develop locally with a tool like **[ngrok](https://ngrok.com/)**.
+
+And here's what the finished app will look like:
+
+![Demo of example app](getting-started-demo.gif)
+
+To make the user flow a bit more explicit:
+1. User 1 initiates a new game and picks their object using the app’s `/challenge` slash command
+2. A message is sent to channel with a button inviting others to accept the challenge
+3. User 2 presses the **Accept** button
+4. User 2 is sent an ephemeral message where they select their object of choice
+5. The result of the game is posted back into the original channel for all to see
+
+## Creating an app
+
+The first thing we’ll need to do is create an app. Navigate to [the developer dashboard](https://discord.com/developers/applications), then click **New Application** in the upper right corner.
+
+![App creation modal](app-create-modal.png)
+
+Enter a name for your app, then click **Create**.
+
+Once you create an app, you'll land on the **General Overview** page of the app's settings. Here you can view and configure basic information about the app, like its description and icon. You’ll also see an **Application ID** and **Interactions Endpoint URL**, which we’ll use a bit later in the guide.
+
+### Configuring a bot
+
+Next we'll add a bot user to your app, which allows it to appear in Discord similar to other members. On the left hand sidebar click **Bot**, then the **Add Bot** button.
+
+Once you create a bot, you’ll have an option to update its icon and username. Under that, there’s a **Token** section with a **Reset Token** button.
+
+![Bot tab in app settings](app-add-bot.png)
+
+Bot tokens are used to authorize API requests and carry all of your bot user’s permissions, making them *highly sensitive*. Make sure to *never* share your token or check it into any kind of version control.
+
+Go ahead and click **Reset Token**, and store the token somewhere safe (like in a password manager).
+
+> warn
+> You won’t be able to view your token again unless you regenerate it, so make sure to keep it somewhere safe. 
+
+### Adding permissions
+
+Apps need approval to perform actions inside of Discord (like installing a slash command or adding emojis). So before installing your app, let's add some permissions. Click on **OAuth2** in the left sidebar, then **URL generator**.
```

Comment:
This seems... a bit strange. "Approval" kind of implies official-ness IMO, that Discord itself needs to grant it

Reply:
Authorization sounds better.